infor-design/enterprise
# 3115- Made flex align bottom work better

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixed two issues with the new field alignment for long labels. As detailed in the steps.

**Related github/jira issue (required)**:
Fixes #3115 

**Steps necessary to review your pull request (required)**:
1. Nesting broke: go to http://localhost:4000/components/grid/test-nesting.html and should work like before like on http://4251-enterprise.demo.design.infor.com/components/grid/test-nesting.html
2. Fixed required errors to keep the alignment: go to http://localhost:4000/components/form/test-long-labels.html and tab out of fields. alignment should be ok even if in error
3. Also rettest #3115 
- go to http://localhost:4000/components/form/example-align-field-bottoms.html
- resize the page and the labels will stay aligned until they drop even if they wrap
- click compact mode and test
- try all themes
- try also http://localhost:4000/components/form/test-wrapped-labels.html this is an older more complex example like example-align-field-bottoms so do the same tests
- go to http://localhost:4000/components/form/test-field-size-data-labels.html and see that the readonly fields align with the text fields
- try compact mode here
- test all the examples in http://localhost:4000/components/form -> should all be working the same
- go to http://localhost:4000/components/form/test-long-labels.html to test the label wrapping on long labels and required fields
